index.markdown

```diff
@@ -621,6 +621,11 @@ Capital One credit card that's directly targeted for people with no credit
 history/newcomers. Beware this card is free for the first year, but costs around
 $30 per year after that.
 
+#### Wells Fargo
+
+Stefan got declined twice trying to get a secured credit card there. Both times they 
+said my SSN was invalid. If you want a CC, wouldn't recommend Wells Fargo.
```

Comment:
Did they read the SSN off your card? Does your SSN have credit history? Do you have credit cards associated with with that SSN?

I know others who've found Wells Fargo really accommodating to Canadian interns.